datasets/lv/magnitsky_list/crawler.py

```diff
@@ -0,0 +1,66 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+# Define constants for date parsing
+DATE_FORMATS = ["%Y-%m-%d", "%Y"]
+
+# Normalized country names
+NORMALIZED_COUNTRIES = {"Krievijas Federācija": "Russia"}
+
+
+def normalize_country(country: str) -> str:
+    """Normalize the country name to a standard format"""
+    return NORMALIZED_COUNTRIES.get(country, country)
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    """Process one row of the CSV data"""
+    original_name = row.get("Original Name")
+    name_in_brackets = row.get("Name in Brackets")
+    birth_date = h.parse_date(row.get("Birth Date"), DATE_FORMATS)
+    country = row.get("Country")
+    source_url = row.get("Source URL")
+
+    if country:
+        normalized_country = normalize_country(country)
+    else:
+        normalized_country = None
+
+    # Create a Person entity
+    entity = context.make("Person")
+    entity.id = context.make_slug(original_name, birth_date)
+
+    # Add names
+    entity.add("name", original_name, lang="lav")
+    entity.add("name", name_in_brackets, lang="eng")
+    # entity.add("alias", name_in_brackets, lang="eng")
+
+    # Add birth date
+    if birth_date:
+        entity.add("birthDate", birth_date)
```

Comment:
Doing `entity.add(prop, None)` is safe (just has no effect) - so you don't need these checks!